MBBSEmu.Tests/Integration/MBBSEmuIntegrationTestBase.cs

```diff
@@ -0,0 +1,95 @@
+using MBBSEmu.DependencyInjection;
+using MBBSEmu.HostProcess;
+using MBBSEmu.IO;
+using MBBSEmu.Module;
+using MBBSEmu.Resources;
+using MBBSEmu.Session;
+using System.Collections.Generic;
+using System.IO;
+using System;
+
+namespace MBBSEmu.Tests.ExportedModules.Majorbbs
```

Comment:
move to MBBSEmu.Tests.Integration namespace and import MBBSEmu.Tests.ExportedModules.Majorbbs